src/components/Person.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+const Person = ({
+  name,
+  age,
+  partnerName,
+  sex,
+}) => {
+  const isWifeOrHusband = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  const hasPartner = partnerName
+    ? `My ${isWifeOrHusband}'s name is ${partnerName}`
+    : <p>I am not married</p>;
+
+  return (
+    <section className="Person">
+      {
+        name && (
```

Comment:
Name is required, don't need conditional rendering.

Reply:
Okay fixed